### PR TITLE
Feat js basics

### PR DESCRIPTION
**Pull request by TaniaBystrova**
js basics homework

### CODE REVIEW

File: git_basics/gitignore
Comment (sttaran) [inline git_basics/gitignore:1]: `.gitignore `file should be placed on the root of the project. 
Path for each entry should be relative
e.g.
`/js_basics//node_modules`

File: git_basics/readme.md
Comment (sttaran) [inline git_basics/readme.md:1]: you should create new branch based on main branch if another condition is not specified

File: js_basics/node_modules/.package-lock.json
Comment (sttaran) [inline js_basics/node_modules/.package-lock.json:1]: node_modules should be ignored by git

File: js_basics/task4.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+let age = 21
+const isAdult = age > 18
```
Comment (sttaran) [inline js_basics/task4.js]: `const isAdult = age >= 18`

File: js_basics/task4.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+let age = 21
```
Comment (sttaran) [inline js_basics/task4.js:1]: use const instead of let

File: js_basics/task3.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+let name1 = 'John'
+let name2 = 'Ann'
```
Comment (sttaran) [inline js_basics/task3.js]: use const instead of let

File: js_basics/task5_1.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
+let radius  = 12
```
Comment (sttaran) [inline js_basics/task5_1.js]: use const instead of let

File: js_basics/task5_2.js [hunk 5]
```diff
@@ -0,0 +1,5 @@
+let length = 10.53
```
Comment (sttaran) [inline js_basics/task5_2.js]: use const instead of let

File: js_basics/task5_3.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+let height = 15
```
Comment (sttaran) [inline js_basics/task5_3.js]: use const instead of let

File: yarn.lock
Comment (sttaran) [inline yarn.lock:1]: this file should be excluded, It has no relation to task

**Review by sttaran — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (TaniaBystrova) [reply yarn.lock:1]: I have deleted this file, what was expected?

**Pull request closed without merging**